pushy/pom.xml

```diff
@@ -96,6 +97,23 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Import-Package>
+                            <!-- Classes from the following packages are only loaded using Class.forName(), so BND can not detect they are required imports. -->
+                            io.netty.channel.socket.nio,
+                            io.netty.channel.socket.oio,
```

Comment:
Looking at this with fresh eyes, I can't imagine any universe in which somebody would use OIO on purpose. Maybe we should just drop support for OIO entirely. That's definitely a separate effort, though.

Reply:
Yes; while it's in the code, it would be best to have it in thee OSGi headers.